tests/test-privacy.php

```diff
@@ -80,4 +95,48 @@ public function test_bookDirectoryExcludedCallback() {
 
 		$this->assertEquals( $buffer, $html_group );
 	}
+
+	/**
+	 * @group privacy
+	 */
+	public function test_getNonCatalogBooks_zero_to_one_non_catalog_books() {
+
+		// assume the first blog is the main wp site and not a book
+		$this->assertIsArray( SharingAndPrivacyOptions::getNonCatalogBooks() );
+		$this->assertCount( 0, SharingAndPrivacyOptions::getNonCatalogBooks() );
+
+		$this->_book();
+		add_site_meta( get_current_blog_id(), 'pb_in_catalog', true );
+
+		$this->assertCount( 1, SharingAndPrivacyOptions::getNonCatalogBooks() );
```

Comment:
Should be `$this->assertCount( 0, SharingAndPrivacyOptions::getNonCatalogBooks() );` I guess

Reply:
revised